build/build.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+echo "######################################################################"
+echo "##### Building AEP Roku SDK"
+echo "######################################################################"
+
+# Get the max number of lines could include license header
+line_max=$1
+
+# Check if the input exists
+if [ -z $line_max ]; then
+    echo "Input is empty."
+    exit 1
+fi
+
+# Create the output directory if not exists
+mkdir -p output
+rm -rf ./output/AEPSDK.brs
+rm -rf ./output/components
+
+# Copy the task node files to output directory
+cp -r ./code/components ./output
+
+# Merge the souce files to one SDK file under output directory
+cat ./code/AEPSDK.brs > ./output/AEPSDK.brs
+
```

Comment:
nit: we should print some steps in build command to know the progress.

Reply:
Let's do it later.